server/src/main/java/org/opensearch/search/SearchHits.java

```diff
@@ -342,4 +399,8 @@ private static Relation parseRelation(String relation) {
             throw new IllegalArgumentException("invalid total hits relation: " + relation);
         }
     }
+
+    public void writeTo(OutputStream out) throws IOException {
```

Comment:
Why is this class not implementing ProtobufWritable? Shouldn't this have been overriden via interface?